pkg/plugins/policies/meshfaultinjection/api/v1alpha1/validator_test.go

```diff
@@ -119,6 +148,77 @@ from:
       - responseBandwidth:
           limit: 1000
           percentage: 1111
+`),
+		ErrorCases("incorrect values with rules",
+			[]validators.Violation{
+				{
+					Field:   `spec.rules[0].default.http.abort[0].httpStatus`,
+					Message: `must be in inclusive range [100, 599]`,
+				},
+				{
+					Field:   `spec.rules[0].default.http.abort[0].percentage`,
+					Message: `must be in inclusive range [0.0, 100.0]`,
+				},
+				{
+					Field:   "spec.rules[0].default.http.delay[1].value",
+					Message: "must not be negative when defined",
+				},
+				{
+					Field:   `spec.rules[0].default.http.delay[1].percentage`,
+					Message: `must be in inclusive range [0.0, 100.0]`,
+				},
+				{
+					Field:   `spec.rules[0].default.http.responseBandwidth[2].responseBandwidth`,
```

Comment:
I think path is missing `limit` here, should be:

```suggestion
					Field:   `spec.rules[0].default.http.responseBandwidth[2].responseBandwidth.limit`,
```

Reply:
fixed